build/bake/file-definition.md

```diff
@@ -349,12 +341,8 @@ $ docker buildx bake "https://github.com/docker/cli.git#v20.10.11" --print
         "VERSION": ""
       },
       "target": "binary",
-      "platforms": [
-        "local"
-      ],
-      "output": [
-        "build"
-      ]
+      "platforms": ["local"],
+      "output": ["build"]
```

Comment:
Maybe prettier wants to fix this but I don't think this is the expected output of `--print`.

Reply:
oops, yep, let me revert this one.